src/SDK/MonitoringApplianceEmulator/Controls/HypertextBlock.cs

```diff
@@ -0,0 +1,135 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="HypertextBlock.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Controls
+{
+    using System;
+    using System.Diagnostics;
+    using System.Text.RegularExpressions;
+    using System.Windows;
+    using System.Windows.Controls;
+    using System.Windows.Documents;
+
+    /// <summary>
+    /// An extension of <see cref="TextBlock"/> control that displays hypertext. The hypertext should be transfered using the <see cref="Hypertext"/> dependency property.
```

Comment:
>An [](start = 8, length = 2)

comment that the control only supports hyperlinks in the format of <a href...>
maybe change the name to HyperlinkTextBlock?

Reply:
Added the comment.  I prefer HypertextBlock because it is a block of Hypertext, not HyperText.
Also, Hypertext is a text that icludes plain text and hyperlinks... so I think it is more acurate as is :)

---
In reply to: [234510281](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/25#discussion_r234510281) [](ancestors = 234510281)